data/virt_autotest/create_bridge.py

```diff
@@ -0,0 +1,403 @@
+# Get the original file from https://gitlab.suse.de/hana-perf/qa_test_hanaonkvm/-/blob/master/kvm_role.py
```

Comment:
What this script can do ? If it aims to create br0 only, give it a more specific name like create_host_bridge.py